_posts/2023-06-26-our-journey-at-f5-with-apache-arrow-part-2.md

```diff
@@ -42,7 +42,7 @@ The following Go Arrow schema definition provides an example of such a schema, i
 var (
   // Arrow schema for the OTLP Arrow Traces record (without attributes, links, and events).
   TracesSchema = arrow.NewSchema([]arrow.Field{
-      // Nullabe:true means the field is optional, in this case of 16 bit unsigned integers 
+      // Nullabe:true means the field is optional, in this case of 16 bit unsigned integers
```

Comment:
🤔  it appears my editor removed trailing spaces automatically as well. I think that is ok